hatchery/helpers.go

```diff
@@ -151,6 +156,8 @@ func getAPIKeyWithContext(ctx context.Context, accessToken string) (apiKey *APIK
 	}
 
 	if resp != nil && resp.StatusCode != 200 {
+		b, _ := ioutil.ReadAll(resp.Body)
```

Comment:
I see that this library `ioutil.ReadAll` is deprecated. [source](https://pkg.go.dev/io/ioutil)

Reply:
it's used in other parts of the code, but i think this is just debug log Jawad left... i think i can remove it